pkgs/build-support/fetchurl/mirrors.nix

```diff
@@ -386,6 +386,10 @@ rec {
   ];
 
   # Python PyPI mirrors
+  
+  pypiio = [
+    https://pypi.io/packages/
+  ];
   pypi = [
```

Comment:
different root base url, the pypi mirror is having a bad starting segment not useful for the different types